src/workloads/execution/MultiPlanning.yml

```diff
@@ -0,0 +1,148 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+ Create collection with multiple indexes and run queries with different selectivity on different
+ indexes to test how efficiently multi planning can fix most selective index.
+
+Keywords:
+- indexes
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  Namespace: &Namespace test.Collection0
+  DocumentCount: &docCount 10000
+  MaxPhases: &MaxPhases 4
+
+ValueGenerators:
+- &Key {^FastRandomString: {length: 6, alphabet: "0123456789"}}
+- &SmallKey {^Join: {array: ["1", {^FastRandomString: {length: 5, alphabet: "0123456789"}}]}}
+- &BigKey {^Join: {array: ["8", {^FastRandomString: {length: 5, alphabet: "0123456789"}}]}}
+- &SelectiveQuery {^Choose: {
+  from: [
+    {$gte: *BigKey},
+    {$lte: *SmallKey}
+  ]
+}}
+- &NonSelectiveQuery {^Choose: {
+  from: [
+    {$gte: *SmallKey},
+    {$lte: *BigKey}
+  ]
+}}
+
+Actors:
+- Name: InitCollectionAndDropPlanCache
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        drop: *Collection
+    - OperationName: RunCommand
+      OperationCommand:
+        createIndexes: *Collection
+        indexes:
+        - key:
+            a: 1
+          name: a_1
+        - key:
+            b: 1
+          name: b_1
+        - key:
+            c: 1
+          name: c_1
+  - &Nop {Nop: true}
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+  - Blocking: None
+    Database: *Database
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        planCacheClear: *Collection
+
+- Name: Insert
+  Type: Loader
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        Threads: 1
+        CollectionCount: 1
+        DocumentCount: *docCount
+        BatchSize: 1000
+        Document:
+          a: *Key
+          b: *Key
+          c: *Key
+
+- Name: Lookup
```

Comment:
I think this could be confused with the $lookup MQL feature. Recommend renaming it something like `MultiPlanning`.

Reply:
Done